dev/before_install

```diff
@@ -20,7 +20,8 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
 	    python3 \
 	    python3-venv \
 	    python3-pip \
-	    nodejs
+	    nodejs \
+	    jq
```

Comment:
just curious...why not in the osx branch?

Reply:
just forget, adding.